pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -69,6 +69,8 @@ const (
 	mxJobFailedReason = "MXJobFailed"
 	// mxJobRestarting is added in a mxjob when it is restarting.
 	mxJobRestartingReason = "MXJobRestarting"
+
+	FailedValidateJobReason = "FailedValidateMXJob"
```

Comment:
Can we use MXJobValidationFailed to follow the convention above?

Reply:
Yes, I agree.